os/arch/Kconfig.chip

```diff
@@ -443,6 +443,13 @@ config ARCH_STACKDUMP
 	---help---
 		Enable to do stack dumps after assertions
 
+config CALLSTACK_WITH_SYMBOL
```

Comment:
This doesn't have showing callstack functionality. This just change an address to symbol. Let's remove "CALLSTACK" from name.